ui/components/multichain-accounts/add-wallet-modal/add-wallet-modal.tsx

```diff
@@ -25,6 +26,15 @@ import {
 import type { ModalProps } from '../../component-library';
 import { FlexDirection } from '../../../helpers/constants/design-system';
 import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  CONNECT_HARDWARE_ROUTE,
+  IMPORT_SRP_ROUTE,
+  ADD_WALLET_PAGE_ROUTE,
+} from '../../../helpers/constants/routes';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../shared/constants/app';
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
```

Comment:
Tried looking for better usage of getEnvironmentType but it looks like it still hasn't been moved to shared